pom.xml

```diff
@@ -1095,7 +1095,7 @@
       <dependency>
         <groupId>org.apache.maven.surefire</groupId>
         <artifactId>surefire-testng</artifactId>
-        <version>2.19</version>
+        <version>3.0.0-M5</version>
```

Comment:
Some comment on why this upgrade was need would be good.

Reply:
I modified pinot-controller.pom to configure the two test suites under pinot-controller. After this, ```mvn test``` would lead to all pinot-controller test cases running twice. Changing configuration settings did not seem to be working, so I tried upgrading to latest version (3.0.0-M5) and that fixed the problem. It seems like the earlier version has a bug that causes testsuites to run twice.

Reply:
Thanks, please add it in the description of the PR, as well as a short description in the pom file itself (can link to the PR description).

Reply:
Done.